front-packages/akeneo-design-system/src/components/Tags/Tags.tsx

```diff
@@ -0,0 +1,109 @@
+import React, {Ref, ReactNode, isValidElement} from 'react';
+import styled from 'styled-components';
+import {AkeneoThemedProps, getFontSize} from 'theme';
```

Comment:
```suggestion
import {AkeneoThemedProps, getFontSize} from '../../theme';
```
I believe this is why the example integration tests are failing https://github.com/akeneo/pim-community-dev/pull/14435/checks?check_run_id=2527324434, we cannot use absolute imports in the DSM 😉